packages/@lwc/ssr-compiler/src/compile-template/transformers/component/slotted-content.ts

```diff
@@ -155,7 +166,10 @@ function getLightSlottedContent(rootNodes: IrChildNode[], cxt: TransformerContex
                     // '' is the default slot name. Text nodes are always slotted into the default slot
                     const slotName =
                         node.type === 'Text' ? b.literal('') : bAttributeValue(node, 'slot');
+                    const { isSlotted } = cxt;
```

Comment:
```suggestion
                    const { isSlotted: originalIsSlotted } = cxt;
```
Since we're unsetting/resetting with different values, using a more descriptive name would make the intention more clear.